src/UI/Art/SpriteArtist/SpriteMapView.java

```diff
@@ -262,6 +262,11 @@ public void render(Graphics g)
       unitArtist.drawUnitIcons(mapGraphics, currentActor, actorCoord.xCoord, actorCoord.yCoord, animIndex);
     }
 
+    for( DamagePopup popup :
```

Comment:
Why move when damage previews are drawn?

Reply:
We can now show damage popups for things that aren't targeted.
I added BBombs as an example.